packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -46,6 +46,7 @@ function webpackConfigSingleSpa(opts) {
       uniqueName: opts.projectName,
       devtoolNamespace: `${opts.projectName}`,
       publicPath: "",
+      jsonpFunction: `webpackJsonp_${opts.projectName}`;
```

Comment:
`jsonpFunction` was removed in webpack 5, and create-single-spa now uses webpack 5. If you're wanting to use webpack 4, use create-single-spa@1.

See https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-unique-naming for the explanation.